net/ip/lwip_mn/src/lwip_socket.c

```diff
@@ -593,6 +593,9 @@ lwip_recvfrom(struct mn_socket *ms, struct os_mbuf **mp,
 #endif
             }
         }
+        if (s->ls_type == MN_SOCK_STREAM) {
```

Comment:
Should make this code conditional on #if LWIP_TCP.

Reply:
done